examples/getST.py

```diff
@@ -318,6 +375,19 @@ def doS4U(self, tgt, cipher, oldSessionKey, sessionKey, nthash, aesKey, kdcHost)
 
         seq_set(authenticator, 'cname', clientName.components_to_asn1)
 
+        reqBodyEncCksumType = cksumtype_for_enctype(cipher.enctype)
+
+        reqBodyEnc = encoder.encode(reqBody)[2:]
```

Comment:
For some reason, the 2 first bytes of the encoded KDC-REQ-BODY sequence are not used for computing the checksum. I am not sure what they represent, but I suppose there is a better way to do this...